src/crm/service/opportunities-service.js

```diff
@@ -0,0 +1,49 @@
+import { HttpClient, json } from 'aurelia-fetch-client';
+import { inject } from 'aurelia-framework';
+
+@inject(HttpClient)
+export class OpportunitiesService {
+  constructor() {
+    this.client = new HttpClient;
+    const baseUrl = '/api/generic/v1/';
+
+    this.client.configure(config => {
+      config.withBaseUrl(baseUrl);
```

Comment:
if you are using DI, please do not change the configuration of the client, since it can affect other services, that need different baseUrl. I suggest using following approach for baseUrl:

```javascript
return this.httpClient
      .fetch(`${this.baseUrl}/new-task`, {
        method: 'post',
        body: body
      })
```